ui/models/filter.js

```diff
@@ -13,8 +13,7 @@ import {
 import { getAllUrlParams } from '../helpers/location';
 
 export default class FilterModel {
```

Comment:
The `history` argument can be removed from the tests now too :-)
https://github.com/mozilla/treeherder/blob/9493b641568466bff62288d5f7471e211d44d471/tests/ui/unit/react/groups.tests.jsx#L16